.github/workflows/ci.yaml

```diff
@@ -612,8 +612,8 @@ jobs:
       SONATYPE_USER: ${{ secrets.SONATYPE_USER_ORGSCALALANG }}
 
     steps:
-      - name: Set JDK 17 as default
-        run: echo "/usr/lib/jvm/java-17-openjdk-amd64/bin" >> $GITHUB_PATH
+      - name: Set JDK 8 as default
+        run: echo "/usr/lib/jvm/java-8-openjdk-amd64/bin" >> $GITHUB_PATH
```

Comment:
Just for making the build future-proof, we should add explicit -source/-release/-target flags to JDK to always emit JDK 8 bytecode and use only JDK API available in that version. 
Same issue might be also present in normal releases

Reply:
I think this would have to be set in SBT, not in the CI configuration. The best we can do from the CI point of view is switch the default Java version.